plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -826,7 +834,7 @@ def get_want_authentication_policy_server(self, auth_policy_server):
                 return self
 
             if len(encryption_key) != 16:
-                self.msg = "encryption_key should be exactly 16 characters."
+                self.msg = "encryption_key must be 16 char long. encryption_key may contain alphanumeric and special characters."
```

Comment:
self.msg = "The 'encryption_key' must be 16 **characters** long. **It** may contain alphanumeric and special characters."